validations/src/tests.rs

```diff
@@ -78,23 +54,21 @@ fn sign_t2<H: Hashable>(tx: &H) -> KeyedSignature {
     }
 }
 
-// Sign with a different public key
-fn sign_t3<H: Hashable>(tx: &H) -> KeyedSignature {
-    let Hash::SHA256(data) = tx.hash();
+static MY_PKH: &str = "wit18cfejmk3305y9kw5xqa59rwnpjzahr57us48vm";
+static MY_PKH_2: &str = "wit1z8mxkml4a50dyysqczsp7gj5pnvz3jsldras8t";
+static MY_PKH_3: &str = "wit164gu2l8p7suvc7zq5xvc27h63td75g6uspwpn5";
+#[test]
+fn test_sign_tx() {
+    let tx = &RevealTransaction::default();
 
-    let secp = &Secp256k1::new();
-    let secret_key =
-        Secp256k1_SecretKey::from_slice(&[0x69; 32]).expect("32 bytes, within curve order");
-    let public_key = Secp256k1_PublicKey::from_secret_key(secp, &secret_key);
-    let public_key = PublicKey::from(public_key);
-    assert_eq!(public_key.pkh(), MY_PKH_3.parse().unwrap());
+    let ks = sign_tx([0xcd; 32], tx);
```

Comment:
I would prefer to use a constant instead of hardcoding the private key:

```suggestion
    let ks = sign_tx(PRIV_KEY_1, tx);
```

And maybe we can rename `MY_PKH` to `MY_PHK_1` to be more consistent